sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppDeploymentImpl.java

```diff
@@ -175,6 +175,56 @@ public List<String> configFilePatterns() {
         }
     }
 
+    @Override
+    public Double cpu() {
+        return Utils.fromCpuString(innerModel().properties().deploymentSettings().resourceRequests().cpu());
+    }
+
+    @Override
+    public Double memoryInGB() {
+        return Utils.fromMemoryString(innerModel().properties().deploymentSettings().resourceRequests().memory());
+    }
+
+    @Override
+    public RuntimeVersion runtimeVersion() {
+        if (isEnterpriseTier() || innerModel().properties() == null) {
```

Comment:
Does EnterpriseTier tell us nothing?

If really so, document it.

Reply:
No, currently they can't give us the deduced version. Will add docs.